paddle/fluid/framework/ir/conv_relu_mkldnn_fuse_pass.cc

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "paddle/fluid/framework/ir/conv_relu_mkldnn_fuse_pass.h"
+#include <string>
+#include <vector>
+#include "paddle/fluid/platform/enforce.h"
+
+namespace paddle {
+namespace framework {
+namespace ir {
+
+void BuildConvReLUPattern(PDPattern* pattern) {
+  // Create Operators
+  auto* conv_op = pattern->NewNode("conv")->assert_is_op("conv2d");
+  auto* relu_op = pattern->NewNode("relu")->assert_is_op("relu");
+  // Create variables
+  // Filter
+  auto* conv_weight_var = pattern->NewNode("conv_weight")
+                              ->AsInput()
+                              ->assert_is_persistable_var()
+                              ->assert_is_op_nth_input("conv2d", "Filter", 0);
```

Comment:
What is the point of the third argument here? It still seems to work if I put there zeros here and in lines 38 and 42.

Reply:
removed